cmd/convox/main.go

```diff
@@ -20,7 +20,12 @@ func init() {
 	stdcli.VersionPrinter(func(c *cli.Context) {
 		fmt.Printf("client: %s\n", c.App.Version)
 
-		system, err := rackClient(c).GetSystem()
+		rc := rackClient(c)
+		if rc == nil {
+			return
```

Comment:
### Current behavior

#### When logged in:

```
$ convox -v
client: 20161118150327
server: 20161111013816 (console.convox.com)
```

#### When not logged in (because convox config dir renamed to something else):

```
$ convox -v
client: 20161118150327
ERROR: must login first
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x53c050]

goroutine 1 [running]:
panic(0x994f00, 0xc420010150)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/convox/rack/client.(*Client).request(0x0, 0xa37758, 0x3, 0xa3aa25, 0x7, 0x0, 0x0, 0x9f6680, 0xc42017f940, 0x32)
	/tmp/release.jHJRd/src/github.com/convox/rack/client/client.go:426 +0x80
github.com/convox/rack/client.(*Client).Get(0x0, 0xa3aa25, 0x7, 0x93f100, 0xc4202c83c0, 0x0, 0x0)
	/tmp/release.jHJRd/src/github.com/convox/rack/client/client.go:46 +0xb1
github.com/convox/rack/client.(*Client).GetSystem(0x0, 0x0, 0xc420213c48, 0x1)
	/tmp/release.jHJRd/src/github.com/convox/rack/client/system.go:25 +0x94
main.init.15.func1(0xc4200a3e00)
	/tmp/release.jHJRd/src/github.com/convox/rack/cmd/convox/main.go:23 +0xd8
github.com/convox/rack/vendor/gopkg.in/urfave/cli%2ev1.ShowVersion(0xc4200a3e00)
	/tmp/release.jHJRd/src/github.com/convox/rack/vendor/gopkg.in/urfave/cli.v1/help.go:166 +0x32
github.com/convox/rack/vendor/gopkg.in/urfave/cli%2ev1.(*App).Run(0xc420138780, 0xc42000c860, 0x2, 0x2, 0x0, 0x0)
	/tmp/release.jHJRd/src/github.com/convox/rack/vendor/gopkg.in/urfave/cli.v1/app.go:204 +0x81b
main.main()
	/tmp/release.jHJRd/src/github.com/convox/rack/cmd/convox/main.go:43 +0xa8
```

### New behavior

#### When logged in:

```
$ convox -v
client: dev
server: 20161111013816 (console.convox.com)
```

#### When not logged in:

```
$ convox -v
client: dev
ERROR: must login first
```